services/task.service.js

```diff
@@ -8,3 +8,13 @@ export const getAllTasks = async () => {
     throw new Error('Error fetching tasks');
   }
 };
+
+export const createTask = async (data) => {
+  try {
+    const newTask = new Task({...data});
+    const savedTask = await newTask.save();
+    return savedTask;
```

Comment:
Also cane empty line before return. It looks more readable.

Reply:
Ok, thanks.